docs/src/main/sphinx/admin/properties-catalog.md

```diff
@@ -0,0 +1,81 @@
+# Catalog management properties
+
+(prop-catalog-management)=
+## `catalog.management`
+
+- **Type:** {ref}`prop-type-string`
```

Comment:
All these links that use 

```
{ref}`prop-type-string`
```

should be changed to use the shorter markdown syntax

```
[](prop-type-string)
```